doc/modules/permutation_importance.rst

```diff
@@ -64,15 +74,44 @@ highlight which features contribute the most to the generalization power of the
 inspected model. Features that are important on the training set but not on the
 held-out set might cause the model to overfit.
 
-.. warning::
-
-  Features that are deemed of **low importance for a bad model** (low
-  cross-validation score) could be **very important for a good model**.
-  Therefore it is always important to evaluate the predictive power of a model
-  using a held-out set (or better with cross-validation) prior to computing
-  importances. Permutation importance does not reflect to the intrinsic
-  predictive value of a feature by itself but **how important this feature is
-  for a particular model**.
+The permutation feature importance is the decrease in a model score when a single
+feature value is randomly shuffled. The score function to be used for the
+computation of importances can be specified with the `scoring` argument,
+which also accepts multiple scorers. Using multiple scorers is more computationally
+efficient than sequentially calling :func:`permutation_importance` several times
+with a different scorer, as it reuses model predictions.
+Different metrics might lead to significantly different feature importances.
+An example of using multiple scorers is shown below, employing a list of metrics,
+but more input formats are possible, as documented in
+`multimetric-scoring <https://scikit-learn.org/stable/modules/model_evaluation.html#multimetric-scoring>`_.
+
+  >>> scoring = ['r2', 'neg_mean_absolute_percentage_error', 'explained_variance']
+  >>> r_multi = permutation_importance(
+  ...     model, X_val, y_val, n_repeats=30, random_state=0, scoring=scoring)
+  ...
+  >>> for metric in r_multi:
+  ...     print(f"{metric}")
+  ...     r = r_multi[metric]
+  ...     for i in r.importances_mean.argsort()[::-1]:
+  ...         if r.importances_mean[i] - 2 * r.importances_std[i] > 0:
+  ...             print(f"    {diabetes.feature_names[i]:<8}"
+  ...             f"{r.importances_mean[i]:.3f}"
+  ...             f" +/- {r.importances_std[i]:.3f}")
+  ...
+  r2
+      s5      0.204 +/- 0.050
+      bmi     0.176 +/- 0.048
+      bp      0.088 +/- 0.033
+      sex     0.056 +/- 0.023
+  neg_mean_absolute_percentage_error
+      s5      0.081 +/- 0.020
+      bmi     0.064 +/- 0.015
+      bp      0.029 +/- 0.010
+  explained_variance
```

Comment:
r2 and explained variance are almost exactly the same. Maybe use `neg_mean_squared_error` instead of explained variance.